presto-native-execution/README.md

```diff
@@ -4,14 +4,55 @@ Prestissimo implements the [Presto Worker REST API](https://prestodb.io/docs/cur
 using [Velox](https://github.com/facebookincubator/velox).
 
 ## Table of Contents
-* [Build from Source](#build-from-source)
+* [Build from Source (conda, new)](#build-from-source-conda-new)
+* [Build from Source (manual)](#build-from-source-manual)
 * [Build using Dockerfile](#build-using-dockerfile)
 * [Development](#development)
 * [Create Pull Request](#create-pull-request)
 * [Advance Velox Version](#advance-velox-version)
 * [Troubleshooting](#troubleshooting)
 
-## Build from Source
+## Build from Source (conda, new)
+
+* Clone the Presto repository:
+
+```bash
+git clone https://github.com/prestodb/presto.git
+```
+
+* Run the following (should work on Linux and macOS):
```

Comment:
```suggestion
2. Run the following (should work on Linux and macOS):
```
Suggest making this unordered list ("Clone the Presto repository:", "Run the following (should work on Linux and macOS):" an ordered list.